calculator/src/test/java/edu/temple/cis3296/CalculatorModelTest.java

```diff
@@ -27,7 +27,12 @@ public void shouldSubstTwoAndTwo(){
 	@Test
 	public void shouldMultTwoByOne(){
 		assertEquals(2.0, calc.operationMult(2.0, 1.0), 0.1);
-	} 
+	}
+
```

Comment:
This looks good. Maybe add a comment to explain what you added, and add another test.